src/main/fc/fc_init.c

```diff
@@ -478,8 +478,8 @@ void init(void)
     initBoardAlignment(boardAlignment());
 
     if (!sensorsAutodetect()) {
-        // if gyro was not detected due to whatever reason, we give up now.
-        failureMode(FAILURE_MISSING_ACC);
+        // if gyro was not detected due to whatever reason, don't arm.
+        setArmingDisabled(ARMING_DISABLED_NO_GYRO);
```

Comment:
The current failureMode code repeats the error code sequence for 10 times... too long to wait if intentionally booting without gyro configured. But yes, it is nice to have a visual indication of the error thinking of the case this happens in the field.

How about
- Move flashing code to, say, `failureLedCode` which takes code repeat count as well as the code,
- `failureMode` will call `failureLedCode` with repeat count = 10,
- `fc_init` can call `failureLedCode` with, say, repeat count = 2.